interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/NodeDataValidatorSpec.scala

```diff
@@ -599,6 +612,25 @@ class NodeDataValidatorSpec extends AnyFunSuite with Matchers with Inside with T
     }
   }
 
+  test(
+    "should validate service based on additional config from provider - P1 as mandatory param with missing actual value"
+  ) {
+    val nodeId = "enricherNodeId"
+    val nodeToBeValidated = node.Enricher(
+      nodeId,
+      ServiceRef("optionalParameterService", List(NodeParameter("optionalParam", Expression.spel("")))),
+      "out"
+    )
+
+    validate(
+      nodeToBeValidated,
+      ValidationContext.empty,
+      outgoingEdges = defaultFragmentOutgoingEdges
+    ) should matchPattern {
+      case ValidationPerformed(List(EmptyMandatoryParameter(_, _, "optionalParam", nodeId)), None, None) =>
```

Comment:
If I understand it correctly, the nodeId variable name in the pattern is not checking if the value is equal to "enricherNodeId" - it's just binding the destructured value to "nodeId" name. It only does the check if the value in the pattern in a literal. This is harmless in this test, but if you look at other tests in this file (even one below) there are unused variables that I think were meant for a check in pattern match.

Reply:
:O you're correct, but that means other tests are also not checking anything (for example "should validate output parameters" and others)